src/test/java/io/debezium/connector/vitess/VitessConnectorIT.java

```diff
@@ -134,8 +134,9 @@ public void shouldOffsetIncrementAfterDDL() throws Exception {
         // apply DDL
         TestHelper.execute("ALTER TABLE numeric_table ADD foo INT default 10;");
         // applying DDL for Vitess version v7.0.1 emits 3 gRPC responses: (VGTID, DDL), (BEGIN, VERSION), (VGTID, COMMIT)
```

Comment:
Shall we remove this comment then?

Reply:
I was hesitating. But I think you're right, no need for this comment. Removed.